### ISSUE TITLE
Train Loss 100多STEP，Loss一直未降

### ISSUE THREAD

**Issue by davidqing2000** [Tianxiaomo/pytorch-YOLOv4#81]
数据标注传入格式如下，使用的是原始图片标注坐标
2007_000027.jpg,174,101,349,351,0 195,180,213,229,3 123,115,379,275,5

但运行了上百个epochs，Loss也未明显下降（附图），是何原因？

![res](https://user-images.githubusercontent.com/6468219/83210491-9629bd80-a18d-11ea-987a-28dafe2358ea.PNG)

**Comment by Tianxiaomo** [comment]
正常，100个lr才多少

**Comment by TWangQing** [comment]
请问这个图是您自己写程序画出来的吗？

> 
> 
> 数据标注传入格式如下，使用的是原始图片标注坐标
> 2007_000027.jpg,174,101,349,351,0 195,180,213,229,3 123,115,379,275,5
> 
> 但运行了上百个epochs，Loss也未明显下降（附图），是何原因？
> 
> ![res](https://user-images.githubusercontent.com/6468219/83210491-9629bd80-a18d-11ea-987a-28dafe2358ea.PNG)

**Comment by davidqing2000** [comment]
不是，Tensorboard 的截图

**Comment by davidqing2000** [comment]
从图形上看上去LOSS值，像是处于随机分布，未见逐步收敛或LOSS下降的趋势。

**Comment by Tianxiaomo** [comment]
lr是多少，batch size大小，用预训练模型了吗，用的哪次提交的代码

**Comment by zengjianyou** [comment]
我也有类似的问题，loss_xy,loss_wh, 震荡不止无法收敛，最后用训练好的模型推理出现nan，没有框

**Comment by markgao-916** [comment]
@Tianxiaomo 我用你硬币数据集训练，按照你提供的batch和subdivisions ，训练到40个eoch，loss还是很大
![图片](https://user-images.githubusercontent.com/50621508/84250215-57591780-ab3e-11ea-8175-7f31f4cb62d1.png)

**Comment by jicongh** [comment]
> @Tianxiaomo 我用你硬币数据集训练，按照你提供的batch和subdivisions ，训练到40个eoch，loss还是很大
> ![图片](https://user-images.githubusercontent.com/50621508/84250215-57591780-ab3e-11ea-8175-7f31f4cb62d1.png)

请问下你训练的硬件设备和训练时间是多少，谢谢？

**Comment by ZhangYang0726** [comment]
batchsize 给小一点就行啦给太大收敛不了的

**Comment by CLemonT** [comment]
我也出现了这种情况，batchsize只设置了2，loss一直都很大。。

**Comment by 606BlueStar** [comment]
> batchsize 给小一点就行啦给太大收敛不了的

请问为什么 batchsize大就收敛不了
